test/docs.jl

```diff
@@ -349,12 +384,7 @@ end
 
 @doc "This should document @m1... since its the result of expansion" @m2_11993
 @test (@doc @m1_11993) !== nothing
-let d = (@doc @m2_11993)
-    @test docstring_startswith(d, doc"""
-    No documentation found.
-
-    `@m2_11993` is a macro.""")
-end
+@test (@doc @m2_11993) === nothing
```

Comment:
Fixed up that case to correctly return the `No documentation found.` message now.